src/blocks/venue/block.json

```diff
@@ -39,5 +39,6 @@
 	"editorStyle": "file:./index.css",
 	"style": "file:./style-index.css",
 	"viewScript": "file:./venue.js",
+	"viewStyle": "file:./venue.css",
```

Comment:
@jmarx all we needed to do was import the leaflet CSS in the OpenStreetMap.js and then add venue.css to viewStyle. No need to important an additional leaflet resource. I think this keeps it clean and simple.

Reply:
@mauteri OK, I guess I have a lot to learn about this stuff. Thanks for fixing.